docs/build-hibench.md

```diff
@@ -48,3 +53,19 @@ Supported modules includes: micro, ml(machine learning), sql, websearch, graph,
 For Spark 2.0 and Spark 2.1, we add the benchmark support for Structured Streaming. This is a new module which cannot be compiled in Spark 1.6. And it won't get compiled by default even if you specify the spark version as 2.0 or 2.1. You must explicitly specify it like this:
 
     mvn -Psparkbench -Dmodules -PstructuredStreaming clean package 
+
+### Build using JDK 1.8
+If you are interested in building using Java 11 specify scala, spark and hadoop version as below
+
+    mvn clean package -Dhadoop=3.2 -Dspark=2.4 -Dscala=2.11
+
+Supported frameworks only: hadoopbench, sparkbench, (Not yet tested flinkbench, stormbench, gearpumpbench)
+Supported modules includes: micro, ml(machine learning), websearch and graph (not tested streaming and structuredStreaming) (Does not support sql)
```

Comment:
I think all modules can be built under JDK8? We normally use 8 in our environment.

Reply:
For Spark 2.4 wont support `sql` benchmarks, Hive not used anymore.
I can be more specific on this and document that for Spark xx version SQL benchmarks not supported.

Reply:
I'm not sure about the `Not yet tested` part, leaving it on master seems... @carsonwang

Reply:
I got rid off it, to avoid causing noise in master.